cluster/src/main/scala/io/snappydata/gemxd/ClusterCallbacksImpl.scala

```diff
@@ -125,4 +124,17 @@ object ClusterCallbacksImpl extends ClusterCallbacks with Logging {
     }
     GemFireVersion.getClusterType
   }
+
+  override def setLeadClassLoader(): Unit = {
```

Comment:
Is it for query execution? If so can't we do at SparkSQLExecuteImpl ?

Reply:
LeadNodeExecutorMsg execute calls the sparksqlexecuteimpl thingy. Just a layer above that.